.github/workflows/pr.yml

```diff
@@ -15,7 +15,7 @@ jobs:
       - name: Checkout source
         uses: actions/checkout@v2
       - name: install
-        run: npm install --no-optional
+        run: ./scripts/dev-env/run.sh ./scripts/setup.sh
```

Comment:
As we build the WT framework as part of the `setup.sh` process, the current state would fail because it cannot find the framework when running `npm install`

Reply:
Is that considered solved?

Reply:
I would suggest we keep this open as a reminder until we've solved the msfs-avionics-mirror hosting question (submodule? build our own? keep it here? strip down? etc. etc.)